libs/core/src/lib/select/select.component.html

```diff
@@ -70,7 +72,7 @@
             [class.is-disabled]="disabled"
             *ngIf="!readonly"
         >
-            <fd-icon [glyph]="glyph"></fd-icon>
+            <fd-icon aria-hidden="true" [glyph]="glyph"></fd-icon>
```

Comment:
I don't think this is applied in the markup. 
<img width="1330" alt="Screenshot 2023-11-07 at 11 50 51 AM" src="https://github.com/SAP/fundamental-ngx/assets/39598672/611839be-c07e-4ebe-91b5-99b1b5fffcf9">